google/cloud/bigtable/admin_client.h

```diff
@@ -80,6 +80,7 @@ class AdminClient {
  protected:
   friend class TableAdmin;
   friend class noex::TableAdmin;
+  friend class PollLongRunningOperation;
```

Comment:
You will need to forward declare the function and declare the function as friend instead, a bit more typing unfortunately.